kubetest2/internal/deployers/eksapi/infra.go

```diff
@@ -85,6 +86,23 @@ func createInfrastructureStack(clients *awsClients, opts *deployerOptions, resou
 		return nil, fmt.Errorf("failed to get infrastructure stack resources: %w", err)
 	}
 	klog.Infof("created infrastructure: %+v", infra)
+
+	if opts.IPFamily == string(ekstypes.IpFamilyIpv6) {
+		for _, subnetID := range infra.subnetsPublic {
+			klog.Infof("enabling auto assign ipv6 on subnet %s", subnetID)
+			_, err = clients._ec2.ModifySubnetAttribute(context.TODO(), &ec2.ModifySubnetAttributeInput{
+				AssignIpv6AddressOnCreation: &ec2types.AttributeBooleanValue{
```

Comment:
Has CFN still not fixed this? Seems like it's in the schema, does it just not work for public subnets?

https://github.com/aws/aws-k8s-tester/blob/main/kubetest2/internal/deployers/eksapi/cloudformation/infra.yaml#L344

Reply:
You tried adding it to the public subnets in the cloudformation template?

Reply:
Good catch. I didn't think of checking if cfn had fixed the issue on their end. 
Update the PR also tested launching a ipv4 and ipv6 cluster, didn't see any issues.